Tests/Library/Client.tests.ts

```diff
@@ -124,13 +124,20 @@ describe("Library/Client", () => {
             trackStub.reset();
             client.trackException(new Error(name));
             client.trackException(new Error(name), properties);
+            client.trackException(new Error(name), properties, measurements);
 
-            assert.ok(trackStub.calledTwice);
+            assert.ok(trackStub.calledThrice);
 
             var args = trackStub.args;
+            
             assert.equal(args[0][0].baseData.exceptions[0].message, name);
+
             assert.equal(args[1][0].baseData.exceptions[0].message, name);
             assert.deepEqual(args[1][0].baseData.properties, properties);
+            
```

Comment:
Should these tests also check that the arguments which were not provided are indeed absent?